friends/src/components/FriendPage.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import EditForm from './EditForm';
+
+const FriendPage = props => {
+    console.log(props, 'test');
+
+    const friendId = props.match.params.id;
+    //console.log(friendId);
+    console.log(props.friends);
+    console.log(friendId);
+    const friend =  props.friends.find(friend => {
```

Comment:
this could have been 
`const friend = props.friends.find(friend => friend.id.toString() === friendId)`